WebRoot/skins/_base/base4/skin.properties

```diff
@@ -2284,3 +2284,22 @@ WhiteBlackListViewRowCell                  = font-weight:normal;
 WhiteBlackListRemoveItemCell               = width:@BigIconWidth@; @ListItemIconWrapper@
 WhiteBlackListViewItemsNumberUsed          = margin-top:@WhiteBlackViewInnerSpacing@;
 
+######################################
+#   OptionsItems ListView
+######################################
+
+OptionsItemsListViewItemsInnerSpacing      = 10px;
+OptionsItemsListColumnHeaderWrapper        = border-left:0px;
+OptionsItemsListColumnHeader               = padding-left:@OptionsItemsListViewItemsInnerSpacing@;
+OptionsItemsListRow                        = overflow:hidden;
+OptionsItemsListLastRow                    = border-bottom:none;
+OptionsItemsListRows                       = overflow:visible;
+OptionsItemsListRowCell                    = padding-left:@OptionsItemsListViewItemsInnerSpacing@; text-align:left;
+OptionsItemsListRowCellSelected            = @OptionsItemsListRowCell@; line-height:1.7rem; white-space:normal;
+
+######################################
+#   Preference -- Zimlet Page
+######################################
+
+PrefZimletDetail                           = margin-bottom:2em; @Text@
+OptionsItemsListItemAlignCenter            = text-align:center; padding-left:0px;
```

Comment:
although property name is not proper but this belongs to block for OptionsItems ListView